packages/@uppy/angular/projects/uppy/angular/package.json

```diff
@@ -25,8 +25,8 @@
     "tslib": "^2.0.0"
   },
   "peerDependencies": {
-    "@angular/common": "^16.2.0",
-    "@angular/core": "^16.2.0",
+    "@angular/common": "^17.3.0",
+    "@angular/core": "^17.3.0",
```

Comment:
How do we know it doesn't work on both? I was wondering if we postponed this for so long while both might have been possible all along.

https://github.com/transloadit/uppy/issues/4893#issuecomment-2009518059

Reply:
Because we don't test it – and having two Angular examples to test this out seems like more work than it's worth.

Reply:
We also don't test 3 React versions but we explicitly support three version ranges. It's not about testing but about expertise about the framework to know how you ship a library for it. What would've been more sensible to me is testing that locally and if it works, we could've just added `|| ^17.0.0` instead of letting users wait five months before we ship a breaking change.